brave/brave-ios
# Fix #3044: Search Suggestion  Text Color blinks when text is entered

This PR fixes Search suggestion Text Color flickering. 

In addition SearchViewController and SearchSuggestionCell are refactored to easy access, better performance and organization.

## Summary of Changes

This pull request fixes #3044

## Submitter Checklist:


## Test Plan:

- Load Brave Browser
- Go Settings - Search Engines - Enable Search Suggestions
- Create a new Tab and load a Webpage and press search bar
- Start typing something
- Observe Search Suggestion Boxes Text color is not flickering from blue to white

## Screenshots:

![ezgif-7-f5203d6961b3](https://user-images.githubusercontent.com/6643505/99704404-58379300-2a66-11eb-9da3-8affae67bea4.gif)

## Reviewer Checklist:

  - `QA/(Yes|No)`
  - `release-notes/(include|exclude)`
  - `bug` / `enhancement`
